usaspending_api/search/v2/views/search.py

```diff
@@ -100,13 +100,16 @@ def post(self, request):
         # Modify queryset to be ordered by requested "sort" in the request or default value(s)
         if sort:
             if subawards:
-                if set(filters["award_type_codes"]) <= set(contract_type_mapping):  # Subaward contracts
+                if set(filters["award_type_codes"]) <= set(list(contract_type_mapping)
+                                                           + list(idv_type_mapping)):  # Subaward contracts
```

Comment:
Why do we add two of the sets but `<=` the third one? I'm asking as someone who genuinely doesn't understand the operation.